core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -827,7 +827,7 @@ private[spark] object ApiHelper {
   def lastStageNameAndDescription(store: AppStatusStore, job: JobData): (String, String) = {
     // Some jobs have only 0 partitions.
     if (job.stageIds.isEmpty) {
-      ("", job.name)
+      ("", "")
```

Comment:
Do we need this change in this PR? UT seems to pass without this.

Reply:
Thanks. I've reverted this change.